pkg/azdiskschedulerextender/azdiskschedulerextender.go

```diff
@@ -395,8 +388,8 @@ func getAzVolumeAttachments(context context.Context, out chan azVolumeAttachment
 	defer Goroutines.WithLabelValues("getAzVolumeAttachments").Dec()
 	// get all azVolumeAttachments running in the cluster
 	var activeVolumeAttachments azVolumeAttachmentsMeta
-	activeVolumeAttachments.volumes, activeVolumeAttachments.err = azVolumeAttachmentInformer.Lister().AzVolumeAttachments(criNamespace).List(labels.Everything())
-	klog.V(2).Infof("AzVolumeAttachments: %d", len(activeVolumeAttachments.volumes))
+	activeVolumeAttachments.volumeAttachments, activeVolumeAttachments.err = azVolumeAttachmentInformer.Lister().AzVolumeAttachments(criNamespace).List(labels.Everything())
+	klog.Infof("AzVolumeAttachments: %v", len(activeVolumeAttachments.volumeAttachments))
```

Comment:
np: We are printing a `len` which is decimal, we don't need to format it as `%v` here. I think we should also keep the logging level as is.

Reply:
Good catch. Not entirely sure how this logging ended up here, probably from the debugging stage when I wanted to check the content of `activeVolumeAttachments`.